product_docs/docs/epas/15/epas_compat_sql/76_update.mdx

```diff
@@ -2,27 +2,41 @@
 title: "UPDATE"
 ---
 
-<div id="update" class="registered_link"></div>
-
 ## Name
 
 `UPDATE` -- update rows of a table.
 
 ## Synopsis
 
-```text
-UPDATE [ <optimizer_hint> ] <table>[@<dblink> ]
-    SET <column> = { <expression> | DEFAULT } [, ...]
-  [ WHERE <condition> ]
-  [ RETURNING <return_expression> [, ...]
+```sql
+edb=# \h UPDATE 
+Command:     UPDATE
+Description: update rows of a table
+Syntax:
+[ WITH [ RECURSIVE ] with_query [, ...] ]
+UPDATE [ <optimizer_hint> ][ ONLY ] <table_name>[@<dblink> ] [ * ] [ [ AS ] alias ]
+    SET { { <column_name> = { <expression> | DEFAULT } |
+            ( <column_name> [, ...] ) = [ ROW ] ( { expression | DEFAULT } [, ...] ) |
+            ( <column_name> [, ...] ) = ( sub-SELECT )
+          } [, ...] |
+          ROW = <row_or_record_var>
+        }
+    [ FROM <from_item> [, ...] ]
+    [ WHERE <condition> | WHERE CURRENT OF <cursor_name> ]
+    [ RETURNING <return_expression> [, ...]
       { INTO { <record> | <variable> [, ...] }
       | BULK COLLECT INTO <collection> [, ...] } ]
+
+URL: https://www.postgresql.org/docs/15/sql-update.html
+     https://www.enterprisedb.com/docs
 ```
 
 ## Description
 
 `UPDATE` changes the values of the specified columns in all rows that satisfy the condition. Only the columns to be modified need be mentioned in the SET clause; columns not explicitly modified retain their previous values.
 
+`SET ROW` clause is available from version 15 onwards. This clause enables us to update a target record using a record type variable or row type objects on the condition that the record or row used should have compatible data types with that table's columns in order.
```

Comment:
- Why are we saying "from version 15 onwards?" I've seen this in a few places. We are in the v15 docs. I see no reason to call out that a change was made in v15. The changes will be captured in the release notes. If you agree combine the first two sentences and and remove "clause is available from version 15 onwards. This clause". Also, search for other instances and make the same type of change.
- who is "us"? are we talking about the user or the product/EDB? If the later I guess "us" is ok. if it is the former use "you"
- the second sentence is really hard to parse. I think you need to add some words and possibly break it into two sentences. It is the "compatible data types with that table's column in order" part that isn't clear.